### PR TITLE
qt is now available for aarch

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
This is ready to be merged @conda-forge/opencv

**Comment by hmaarrfk** [comment]
is this really a feature ;) I'm only asking because qt is so large of a download. Is anybody actually using Qt + opencv on an arm processor today?

Do *you* want this to be merged beyond the intellectual curiosity of building the package?

**Comment by Tobias-Fischer** [comment]
Yes please - some packages that we use in RoboStack pull in some deps that are only available if OpenCV is compiled with Qt (I can't remember exactly which ones, but this PR is fixing https://github.com/conda-forge/opencv-feedstock/issues/245 that I presumably opened for a good reason ;))

File: recipe/meta.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ requirements:
     - harfbuzz                       # [unix]
     # Don't depend on python in the run section
     # py-opencv will depend on python
-    - qt 5.12                        # [not osx and not aarch64 and not ppc64le]
+    - qt 5.12                        # [not osx and not ppc64le]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:107]: why do we have this strange pin again?

**Pull request merged**